diagnostic.md

```diff
@@ -6,70 +6,74 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What are two ways of telling rails where to route a GET request to '/books/4'?
 
 ```rb
-// your response here
+get '/books', to: 'books#index'
+resources :books, only: [:index]
 ```
 
 In what file does the above code go?
 
 ```md
-// your response here
+routes.rb
 ```
 
 In your own words, what does the command `bundle exec rake db:migrate` do?
 
 ```md
-// your response here
+this command executes the rake script with the command db:migrate in the context
+
+
 ```
 
 What is the difference between `rails g scaffold things` and
 `rails g controller things`?
 
 ```md
-// your response here
+rails g controller with generate just a controller shell while rails g
+scaffold will generate shells for model, view and controller
 ```
 
 What is the job of the `serializer`?
 
 ```md
-// your response here
+allows you to store almost any object in a single database field
```

Comment:
Th serializer allows you to filter certain data out of your response. The data still exists, it's just not shown in the response to users.